CairoMakie/Project.toml

```diff
@@ -22,7 +22,7 @@ FFTW = "1"
 FileIO = "1.1"
 FreeType = "3, 4.0"
 GeometryBasics = "0.4.1"
-Makie = "=0.17"
+Makie = "=0.17.1"
```

Comment:
Is this necessary?  As I understand it, forcing Makie to `"^0.17.1" in AlgebraOfGraphics should be sufficient?  It seems a bit strict to me, especially since the new version of CairoMakie should be compatible with Makie 0.17.

Reply:
I think that's just how it's being done since the monorepo: the backends pick a specific version of Makie, see eg [here](https://github.com/JuliaPlots/Makie.jl/blob/v0.16.6/CairoMakie/Project.toml).

Reply:
Makes sense.  LGTM!